pyproject.toml

```diff
@@ -13,7 +13,6 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = ">=3.8,<3.9.7 || >3.9.7,<4.0"
 torch = "^1.13.1"
-openai-whisper = "^20230124"
```

Comment:
I used `poetry` just to avoid the dependencies conflict, but it seems like it didn't get them right at the end.

I agree with you, let's just keep whisper-timestamped only. 
Hope we won't find any issues in the long run.